Since fc14753dc0188a52a05243d5d82c4062b93daaff
the wine virtual desktop window doesn't have a title anymore.
This commit fixes this.

This is needed because DesktopWndProc ignores the WM_SETTEXT message.
Alternative would be handling WM_SETTEXT in DesktopWndProc.

Signed-off-by: Fabian Maurer <dark.shadow4 at web.de>
---
 programs/explorer/desktop.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/programs/explorer/desktop.c b/programs/explorer/desktop.c
index 4f47ad0d36b..97bc9e40a6b 100644
--- a/programs/explorer/desktop.c
+++ b/programs/explorer/desktop.c
@@ -662,6 +662,9 @@ static LRESULT WINAPI desktop_wnd_proc( HWND hwnd, UINT message, WPARAM wp, LPAR
             EndPaint( hwnd, &ps );
         }
         return 0;
+
+    case  WM_SETTEXT:
+        return DefWindowProcW( hwnd, message, wp, lp );
     }
 
     return desktop_orig_wndproc( hwnd, message, wp, lp );
-- 
2.18.0
